liquibase-core/src/main/java/liquibase/database/DatabaseFactory.java

```diff
@@ -291,23 +294,26 @@ private Properties buildDriverProperties(String username, String password, Strin
                 driverProperties.put("password", password);
             }
             if (null != driverPropertiesFile) {
-                File propertiesFile = new File(driverPropertiesFile);
-                if (propertiesFile.exists()) {
-                    LOG.fine(
-                            "Loading properties from the file:'" + driverPropertiesFile + "'"
-                    );
-                    FileInputStream inputStream = new FileInputStream(propertiesFile);
-                    try {
-                        driverProperties.load(inputStream);
-                    } finally {
-                        inputStream.close();
+                try {
+                    String driverPropertiesFilePath = resourceAccessor.resolve(null, driverPropertiesFile);
```

Comment:
You'll want to use the PathHandlerFactory in here as well too. In the soon-incoming `add-resource` branch there is a single call that will do this logic of looking for a file first via the way it's specified (including s3 etc.) and then falling back to the resource accessor as needed.

It's maybe best to wait for that PR to get through instead of doing it this way and then changing it for that?